### DISCUSSION TITLE
Support for Pgenerator

### DISCUSSION THREAD

**Discussion by lillgunde** [eoyilmaz/displaycal-py3#290] [Ideas]
A great future feature for the modernization of Displaycal would be Pgenerator support.

Also,
When using the "Auto-Optimized" selection under the profiling tab there is no way of knowing what
patches are being sent to the renderer/patch generator.

It would be nice if you could specify what output the software is sending the pattern generator.
REC.709 /D65 , BT2020 /D65, BT2020 P3/65 etc. and adjust the patterns accordingly.

**Comment by eoyilmaz** [comment]
I believe you can condition the first set of patches to be picked from a certain gamut so that instead of randomly firing patches Argyll generates patches from a known gamut, which I believe more or less matches your request.

To do that:
1. Go the profiling tab
2. In the Profile type section, select the cog wheel icon:
![Screenshot 2024-03-28 at 10 03 52](https://github.com/eoyilmaz/displaycal-py3/assets/1786804/4148b6e8-8277-49f2-8c1a-7a9a70005f45)
3. Select the profil from the source profile section
![Screenshot 2024-03-28 at 10 06 40](https://github.com/eoyilmaz/displaycal-py3/assets/1786804/c6d931ad-f274-4654-b321-5ac640f1a3cb)

P.S: Just to let you know, there is a bug related to that dialog, I'll create a bug report for that.
